### DISCUSSION TITLE
"Allow extra latency to improve performance" option

### DISCUSSION THREAD

**Discussion by rampaa** [Blinue/Magpie#655] [Q&A] [answered]
What does this option exactly do? Does enabling it for scenarios mentioned in https://github.com/Blinue/Magpie/discussions/610 make sense?

**Comment by Blinue** [comment, marked as answer]
By default, Magpie tends to reduce latency, which sometimes can significantly lower the frame rate. Turning on this option can make full use of the GPU, but it will introduce one frame of extra latency. #610 has nothing to do with this option.
